### DISCUSSION TITLE
Application errors when MinIO user and password are specified

### DISCUSSION THREAD

**Discussion by aalyuza** [Budibase/budibase#13162] [Help]
Hello.
Budibase is launched on server using Portainer.
According to the [manual](https://docs.budibase.com/docs/accessing-minio), i specified the environment parameters MINIO_BROWSER, MINIO_ACCESS_KEY and MINIO_SECRET_KEY and configured port forwarding.
The MinIO client is working, but Budibase applications are no longer running. Error "Error initialising app:".
If I add a new user in the MinIO client and remove the environment settings MINIO_ACCESS_KEY and MINIO_SECRET_KEY, then I will have access to the MinIO and the Budibase applications will work.
Could I have done something wrong?

**Comment by marceldarvas** [comment]
I am having the same issue.
